src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/OneTopLevelClass/patchedline/defaultContext.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/Test.java b/Test.java
+index f7fa153..92449e0 100644
+--- a/Test.java
++++ b/Test.java
+@@ -6,3 +6,7 @@ public class Test {
+ class Test2 {  // violation without filter
```

Comment:
Please apply a change to this line both in newline strategy and patched line strategy. Update expected results correspondingly.

Reply:
done